components/StoreProducts.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { View } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+
+import { Button, Title } from '../styles/shared';
+import Product from './Product';
+import StoreCard from './StoreCard';
+
```

Comment:
@anniero98 since these categories are hard-coded and reused throughout this file -- let's make them global constants?

```
const CATEGORY  = {
FRUIT: 'fruit',
VEGETABLES: 'vegetables'
} 
const NAV = {
STORES_DETAILED = 'StoresDetailed',
PRODUCTS = 'products`
}
```

Reply:
We'll address this in another PR!